app/src/models/init-models.ts

```diff
@@ -748,6 +764,12 @@ export function initModels(sequelize: Sequelize) {
     as: "assistantMessages",
     foreignKey: "threadId",
   });
+  Project.belongsTo(Organization, {
+    as: "organization",
+    foreignKey: "organizationId",
+  });
```

Comment:
Organization.hasMany(Project, ...) seems to be missing?

Reply:
because the Organization model is in my PR but not in Isasac's. We'll have to do a bit of cleanup once one of the PRs is merged